variants/vmware-k8s-1.19/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+# This is the vmware-k8s-1.19 variant.  "." is not allowed in crate names, but
+# we don't use this crate name anywhere.
+name = "vmware-k8s-1_19"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant]
+image-format = "vmdk"
+included-packages = [
+    "cni",
+    "cni-plugins",
+    "kubelet-1.19",
+    "release",
+    "static-pods",
```

Comment:
`static-pods` should not be listed, that's handled by the `os` package now.

Reply:
Ah - good catch.